### PR TITLE
chore: remove empty buckets spinner

### PR DESCRIPTION
**Pull request by CaptainCarpensir**
Removes the spinner for emptying buckets. This is because most applications won't actually have any s3 buckets to delete, and showing this is highly misleading to users.

Follow up to #5184 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/cli/env_delete.go [hunk 1]
```diff
@@ -181,19 +181,14 @@ func (o *deleteEnvOpts) Execute() error {
...
-		// Handle error and recommend action, don't exit program
-		switch t := err.(type) {
-		case *errBucketEmptyingFailed:
-			log.Errorln(t.Error())
-			log.Warningln(t.RecommendActions())
-		default:
-			log.Errorln(fmt.Errorf("empty s3 buckets: %w", err))
+		// Handle empty bucket error and recommend action, don't exit program. Otherwise swallow error and move on.
+		var emptyBucketErr *errBucketEmptyingFailed
+		if errors.As(err, &emptyBucketErr) {
```
Comment (dannyrandall) [inline internal/pkg/cli/env_delete.go:188]: do we not care about the case where `!errors.As(err, &emptyBucketErr)` anymore?

Comment (CaptainCarpensir) [reply internal/pkg/cli/env_delete.go:188]: Yes and no. Any errors that can happen here other than `emptyBucketErr` are likely gonna be the result of another issue outside of the s3 buckets. However in same cases it could be. This failing should never block the deletion of buckets without showing an error though.

My concern is with the case where one of these unrelated errors isn't obvious, and we swallow it here. I don't think this will be the case but frankly there's a ton of ways for the various API calls it makes to fail and it could potentially block something. The problem is that the alternative is that we error out for an unnecessary call when they don't have access logs enabled. It then means we display an error for something that's entirely non-blocking, but this function can't know if access logs are enabled on call, it finds out in the function.

TLDR in most cases swallowing the error is the right call. We take the trade-off of swallowing a potentially blocking error in order to not confuse the user with an error likely originating from a different source.

Comment (dannyrandall) [reply internal/pkg/cli/env_delete.go:188]: totally on board with not blocking - but we used to print out the error, and now we don't print it out unless it's `errBucketEmptyingFailed`🤔

Comment (dannyrandall) [reply internal/pkg/cli/env_delete.go:188]: oh i see what you're saying 👍 sounds good

**Review by dannyrandall — APPROVED** [review]

**Review by iamhopaul123 — APPROVED** [review]
